packages/flutter_test/lib/src/test_pointer.dart

```diff
@@ -260,13 +314,27 @@ class TestGesture {
 
   /// In a test, send a move event that moves the pointer by the given offset.
   @visibleForTesting
-  Future<void> updateWithCustomEvent(PointerEvent event, { Duration timeStamp = Duration.zero }) {
+  Future<void> updateWithCustomEvent(PointerEvent event, {Duration timeStamp = Duration.zero}) {
```

Comment:
`{ Duration timeStamp = Duration.zero }` I think I mentioned this earlier. Flutter sources generally include the spaces before named parameters.